kong/core/balancer.lua

```diff
@@ -603,10 +650,10 @@ end
 
 
 local function init()
+
   local upstreams, err = get_all_upstreams()
   if not upstreams then
-    log(ngx.STDERR, "failed loading initial list of upstreams: ", err)
-    return
+    return nil, "failed loading initial list of upstreams: " .. err
```

Comment:
It seems we don't log this returned error in our `balancer.init()` call in `init_worker`? Maybe we should keep this as a `CRIT` log + early return?

Reply:
ok! 👍